security-header/README.md

```diff
@@ -4,7 +4,22 @@ Get an **A+** on [security header](https://securityheaders.com/) out of the box.
 
 Before I was using CloudFlare worker (5$ per month) to get the same result. I asked myself, how does the ‘real server’ are configured to manage the scurity-header? This is how.
 
-**screenshot**:
+# Warning
+
+Test those default values locally as they could prevent your website functioning normally.
+
+- Content-Security-Policy	Content Security Policy is an effective measure to protect your site from XSS attacks. By whitelisting sources of approved content, you can prevent the browser from loading malicious assets. Analyse this policy in more detail. You can sign up for a free account on Report URI to collect reports about problems on your site.
```

Comment:
Could we format these bullet items a little nicer? The extra spacing doesn't do anything in reality and it all runs together. Try bolding instead.